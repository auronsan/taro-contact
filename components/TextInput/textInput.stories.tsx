import React from 'react';
import { TextInput } from './index';

export default {
  title: 'components/TextInput',
  component: TextInput,
};

export const Default = () => (
  <TextInput id="default-text-input" name="default" placeholder="Default Text Input" />
);

export const WithLabel = () => (
  <TextInput
    id="labeled-text-input"
    name="labeled"
    label="Labeled Text Input"
    placeholder="Text Input"
  />
);
