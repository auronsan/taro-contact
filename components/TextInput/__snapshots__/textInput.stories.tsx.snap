// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/TextInput Default smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <div class="textinput_root__IJOrE">
      <input class="textinput_input__ICHQz"
             data-testid="default-text-input"
             id="default-text-input"
             placeholder="Default Text Input"
             name="default"
      >
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`components/TextInput WithLabel smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <div class="textinput_root__IJOrE">
      <label class="textinput_label__Fgwby"
             for="labeled-text-input"
      >
        Labeled Text Input
      </label>
      <input class="textinput_input__ICHQz"
             data-testid="labeled-text-input"
             id="labeled-text-input"
             placeholder="Text Input"
             name="labeled"
      >
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
